### ISSUE TITLE
Increasing line height does not scale paragraph gaps

### ISSUE THREAD

**Issue by detjensrobert** [creesch/readReddit#4]
This might just be inherent to the way spacing works, but increasing the `Line height` option in the extension settings does not increase the height of the space between paragraphs.

Closer spacing (1.4 em):
![alt text](https://img.ourl.ca/large.png "Closer spacing")

Larger spacing (1.8 em):
![alt text](https://img.ourl.ca/small.png "Larger spacing")

Firefox 67, latest version of readReddit

**Comment by creesch** [comment]
That's correct line height and the margin between paragraphs are two different things. I suppose I also could add an option for paragraph margin though.
